features/project/project_controller.rb

```diff
@@ -12,6 +12,8 @@ class ProjectController < AuthenticatedControllerBase
     # TODO: this should actually be a POST request
     get "#{HOME}/:project_id/trigger" do
       project_id = params[:project_id]
+      lane = params[:lane] # TODO: not sent yet
+      platform = params[:platform] # TODO: not sent yet
```

Comment:
right now we get `lane` from the project, but we don't store `platform` maybe we need to add this to the `Project` model